src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-wrapper">
+      <form>
+        <div
+          class="search-form search-form--big"
+          data-qa="big"
+        >
+          <input
+            type="text"
+            placeholder="Try 'Los Angeles'"
+            data-qa="keypress"
+          />
```

Comment:
Consider adding a `name` attribute to the input element for better form data handling and accessibility. This attribute is important for identifying form data when submitted.